packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -978,3 +942,19 @@ function mapBusinessPlatformStoresToOrganizationStores(storesArray: ShopNode[]):
     } as OrganizationStore
   })
 }
+
+function appModuleVersion(mod: ReleasedAppModuleFragment): Required<AppModuleVersion> {
+  return {
+    registrationId: mod.uuid,
```

Comment:
non-blocking on this: but I would still like us to sort this particular thing out -- lots of IDs in the result format, non-obvious matching from API to result format, & app-management _not_ being the default behaviour (currently partners is setting the pattern)

Reply:
Only Partners needs ID, with the result that eventually we will be able to drop ID and just retain UUID. (Title is not an immutable ID, but a mutable display concern.) I'm not really very worried about that now, and I don't think it's worth investing in further cleaning this up, when we'll be refactoring code just to delete it.